src/ValueObject/Week.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\DpdPlPickupServicesBundle\ValueObject;
+
+use Answear\DpdPlPickupServicesBundle\Enum\Day;
+use Webmozart\Assert\Assert;
+
+class Week implements \IteratorAggregate
```

Comment:
To me, the name doesn't really tell you what it is about. Maybe it should be more descriptive?

Reply:
Do suggest :)